dask_sql/datacontainer.py

```diff
@@ -161,7 +161,11 @@ def get_backend_by_frontend_name(self, column: str) -> str:
             # Make sure to get the backend column name here by using [1] instead of [0]. [0] is frontend
             backend_column = list(self._frontend_backend_mapping.items())[0][1]
         else:
-            backend_column = self._frontend_backend_mapping[column]
+            try:
+                backend_column = self._frontend_backend_mapping[column]
+            except KeyError:
+                backend_column = column[column.find("(")+1:column.find(")")]
```

Comment:
For the `SUM(2) AS "X"` example, `column` is "Int64(2)", so this line of code is extracting 2 from the parentheses.

Reply:
cc: @charlesbluca since some of the things you are looking into may/may not handle this case